extension.toml

```
@@ -11,6 +11,11 @@ name = "Starpls"
 language = "Starlark"
 languages = ["Starlark"]
 
+[language_servers.buck2-lsp]
```

Maybe the id should just be `buck2`? Is there a good reason to append `-lsp`?

```suggestion
[language_servers.buck2]
```

---

Doesn't hurt to be extra clear, just to ensure no confusion propagates to the end user.